### PR TITLE
Add weak dependencies (remade)

### PR DESCRIPTION
**Pull request by nexfwall**
Add weak dependencies for plugin and main packages.

https://fedoraproject.org/wiki/Packaging:WeakDependencies

### CODE REVIEW

**Comment by nexfwall** [comment]
Package with this patch was successfully built for Rawhide and Fedora 24 using mockchain.

File: mpg123.spec [hunk 1]
```diff
@@ -20,6 +20,8 @@ BuildRequires:  pkgconfig(alsa)
 
 Requires:       %{name}-libs%{?_isa} = %{?epoch:%{epoch}:}%{version}-%{release}
 
+Recommends:     %{name}-plugins-pulseaudio%{?_isa} = %{?epoch:%{epoch}:}%{version}-%{release}
```
Comment (ignatenkobrain) [inline mpg123.spec]: EVR is not needed here as `%{name}-plugins-plulseaudio` explicitly requires main package with EVR.

**Comment by ignatenkobrain** [comment]
I think correct way for pulseaudio plugin and others to use Rich dependencies.

**Comment by nexfwall** [comment]
Package with this changes was successfully built on mock.

**Comment by ignatenkobrain** [comment]
merged with few fixups.

**Pull request closed without merging**